safe_transaction_service/tokens/tasks.py

```diff
@@ -111,21 +111,3 @@ def fix_pool_tokens_task() -> Optional[int]:
         if number:
             logger.info("%d pool token names were fixed", number)
         return number
-
-
-@app.shared_task()
-@close_gevent_db_connection_decorator
-def get_token_info_from_blockchain_task(token_address: ChecksumAddress) -> bool:
```

Comment:
@Uxio0 your review here is appreciated 🙂 I removed the task completely but I'm not sure the cache management is correct. It seems this function also stores an `int` to indicate the token was retrieved (to avoid requesting it several time if I understand correctly), so I suppose the actual caching (if it's being cached) of the value is done outside this function 🤔 so it should be OK to remove this code completely.

Reply:
No worries, it shouldn't have any conflicts. If we need that function we can just revert or just write it again